### ISSUE TITLE
[Accessibility] Search button in a compact Toolbox doesn't have an accessible text

### ISSUE THREAD

**Issue by OlgaLarina** [surveyjs/survey-creator#6953]
Ensure buttons have discernible text

![Image](https://github.com/user-attachments/assets/58d9764e-36d7-44a3-bf36-87555f678e62)
